### PR TITLE
Docs/osb docs changes

### PR DESCRIPTION
**Pull request by junior**
Documentation changes supporting PR #81

### CODE REVIEW

File: src/docs/source/includes/topics/_grafana.md [hunk 1]
```diff
@@ -6,35 +6,35 @@ Revisit the application charts and connect to some Grafana dashboards:
...
+    mushop                  mushop                  1               2020-01-31 21:14:48.511917 -0600 CST    deployed        mushop-0.1.0                    1.0         
+    mushop-utility          mushop-utilities        1               2020-01-31 20:32:05.864769 -0600 CST    deployed        mushop-setup-0.0.1              1.0          
     ```
 
-1. Get the Grafana outputs from the `mushop-setup` installation:
+1. Get the Grafana outputs from the `mushop-utility` (setup chart) installation:
 
     ```text
-    helm status mushop-setup
+    helm status status mushop-utility --namespace mushop-utilities
```
Comment (mvandervliet) [inline src/docs/source/includes/topics/_grafana.md]: dupe `status`

File: src/docs/source/includes/_deployment.md [hunk 2]
```diff
@@ -201,17 +211,17 @@ values as described [above](#provisioning)
...
+      --name mushop-osb \
       --set ociCredentials.secretName=oci-credentials \
       --set storage.etcd.useEmbedded=true \
       --set tls.enabled=false
     ```
 
     ```shell--helm3
-    helm install oci-service-broker https://github.com/oracle/oci-service-broker/releases/download/v1.3.2/oci-service-broker-1.3.2.tgz \
-      --namespace mushop \
+    helm install mushop-osb https://github.com/oracle/oci-service-broker/releases/download/v1.3.2/oci-service-broker-1.3.3.tgz \
```
Comment (mvandervliet) [inline src/docs/source/includes/_deployment.md]: ~v1.3.2~

File: src/docs/source/includes/_deployment.md [hunk 3]
```diff
@@ -201,17 +211,17 @@ values as described [above](#provisioning)
 1. Deploy the OCI service broker on your cluster. This is done with the [Oracle OCI Service Broker](https://github.com/oracle/oci-service-broker) helm chart:
 
     ```shell--helm2
-    helm install https://github.com/oracle/oci-service-broker/releases/download/v1.3.2/oci-service-broker-1.3.2.tgz \
-      --namespace mushop \
-      --name oci-service-broker \
+    helm install https://github.com/oracle/oci-service-broker/releases/download/v1.3.2/oci-service-broker-1.3.3.tgz \
```
Comment (mvandervliet) [inline src/docs/source/includes/_deployment.md]: ~v1.3.2~

File: src/docs/source/includes/_cleanup.md [hunk 4]
```diff
@@ -29,12 +31,32 @@ depending on the actions performed for setup and deployment of MuShop.
...
+
+    ```shell--helm3
+    helm delete mushop-osb
+    ```
+
 - Remove the `setup` cluster dependency installation:
 
     ```shell--helm2
-    helm delete --purge mushop-setup
+    helm delete --purge mushop-utility
```
Comment (mvandervliet) [inline src/docs/source/includes/_cleanup.md]: Suggestion: use `utils` and `utilities` for separation. plural in both cases

**Comment by junior** [comment]
> Suggest the following changes:
> 
> * Rename `mushop-osb` to either `oci-service-broker` or simply `osb`
> * Use `mushop-utils` as the release name for `setup` chart (to keep plural)
> * Add notes that the `oci-service-broker-1.3.3.tgz` chart **MUST** be in the same namespace as the `setup` chart installation. This can be an `<aside class="warning">...</aside>`

_* Use `mushop-utils` as the release name for `setup` chart (to keep plural):_ 
---Changed as requested with new commit. Note: There's no Helm guidelines for naming, just the recommendation to independent from the name of the chart. With Helm3, the release is stored to a specific namespace, and is becoming best practice to not repeat the name of the chart and show a clear separation. It's also best practice to include the name of the app consuming the supporting service on the release name for better identification, but again, there's no specific guideline for that. The most used helm charts defaults the name of the release in the singular, but there's no guidelines around that. Documentation and Slides changed from mushop-utility to mushop-utils.

_* Add notes that the `oci-service-broker-1.3.3.tgz` chart **MUST** be in the same namespace as the `setup` chart installation. This can be an `<aside class="warning">...</aside>`:_  
---New commit includes the warning.

 _* Rename `mushop-osb` to either `oci-service-broker` or simply `osb`:_ 
---This follows on the same recommendation above. Also to avoid the pods to be named **oci-service-broker-oci-service-broker-xptoxyz**. I think that can be any name, just should be more informative and granular and not repeat the name of the app to the name of the chart to the name of the release. `osb-broker` (or `osb-brokers`) works for you? 



Extra Notes: For the OCI Service Broker specifically, you can have more than one broker on the same installation and in production you can even have more than one version, installed in a separate release. The idea of Release Names are exactly to help this.

**Comment by mvandervliet** [comment]
> @mvandervliet `osb-broker` (or `osb-brokers`) works for you?
The point is that this is specific to the open source `oci-service-broker` rather than mushop (eve though mushop is using). So, with that in mind, and the intentional differentiation between _release name_ and _chart name_, I think `oci-broker` is clearest to users (singular because this is just one chart, unlike `mushop-utils` containing multiple things)

**Comment by junior** [comment]
> @mvandervliet `osb-broker` (or `osb-brokers`) works for you?
> The point is that this is specific to the open source `oci-service-broker` rather than mushop (eve though mushop is using). So, with that in mind, and the intentional differentiation between _release name_ and _chart name_, I think `oci-broker` is clearest to users (singular because this is just one chart, unlike `mushop-utils` containing multiple things)

Last commit changed from mushop-osb to oci-broker. 

Some remarks still valid for when install more that one broker on the same cluster. Before was `mushop-osb` -> Prod, `mushop-staging-osb` -> staging, now can be something like`oci-broker-staging` if needed. The release name currently direct affects how communicate to the oci-service-broker.

**Pull request merged**